### PR TITLE
VZ 4383: Dump certs + secrets (no .data) 

### PR DESCRIPTION
**Pull request by anders-swanson**

### CODE REVIEW

File: tests/e2e/examples/socks/sock_shop_example_test.go [hunk 1]
```diff
@@ -245,66 +245,65 @@ var _ = Describe("Sock Shop Application", func() {
...
+				_, err := pkg.GetNamespace(sockshopNamespace)
+				if err != nil && errors.IsNotFound(err) {
+					return true
+				}
+				if err != nil {
+					pkg.Log(pkg.Info, fmt.Sprintf("Error getting sockshop namespace: %v\n", err.Error()))
+				}
+				return false
+			}, shortWaitTimeout, shortPollingInterval).Should(BeTrue())
+		})
```
Comment (pfmackin) [inline tests/e2e/examples/socks/sock_shop_example_test.go:298]: Where do you dump the cluster if the cleanup fails?

File: tests/e2e/examples/socks/sock_shop_example_test.go [hunk 2]
```diff
@@ -245,66 +245,65 @@ var _ = Describe("Sock Shop Application", func() {
...
 	})
 
+	if failed {
+		It("Should dump cluster if previous specs have failed", func() {
+			err = pkg.ExecuteClusterDumpWithEnvVarConfig()
+			Expect(err).To(BeNil())
+		})
+	}
+
+	if !skipUndeploy {
```
Comment (pfmackin) [inline tests/e2e/examples/socks/sock_shop_example_test.go:255]: Will this always run after the other tests?

File: tests/e2e/examples/socks/sock_shop_example_test.go (hunk 1 above)
Comment (anders-swanson) [reply tests/e2e/examples/socks/sock_shop_example_test.go:298]: Not sure. How do we explicitly set this?

File: tests/e2e/examples/socks/sock_shop_example_test.go (hunk 2 above)
Comment (anders-swanson) [reply tests/e2e/examples/socks/sock_shop_example_test.go:255]: No, skipUndeploy is a flag variable. See -> https://github.com/verrazzano/verrazzano/blob/master/tests/e2e/examples/socks/sock_shop_example_suite_test.go#L18

Comment (anders-swanson) [reply tests/e2e/examples/socks/sock_shop_example_test.go:255]: if you mean will ginkgo run these cases in order, I don't know.

File: tests/e2e/examples/socks/sock_shop_example_test.go (hunk 1 above)
Comment (kminder) [reply tests/e2e/examples/socks/sock_shop_example_test.go:298]: Where the call to executeClusterDumpWithEnvVarConfig if undeployment or namespace deletion fails?

**Review by pfmackin — APPROVED** [review]

**Review by kminder — APPROVED** [review]
Approved

**Pull request merged**